hack/install-operator-sdk.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
```

Comment:
The goal of this line is to get the absolute path of the script dir. We are not really changing the dir. Reasons to use the current method:
1. Using BASH_SOURCE[0] ensures that we get the right dir even if the script is sourced in a different script. This is of less concern for this script. This is handles the case where the script is called through a symbolic link.
2. cd and pwd together give us the absolute path instead of the relative path.

Although cryptic, this is more fool proof way to accomplishing the task. Considering that we don't use symbolic links and sourcing of the script. I will simplify it to 
```
SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
```